src/main/java/com/digitax/controller/AuthController.java

```diff
@@ -405,7 +408,12 @@ else if (userRepository.existsByUsername(loginRequest.getUsername())) {
         }
     }
     
-    
+    /**##
+     * 
```

Comment:
@Jayanta-Redapple There are no descriptions added to your javadocs. Add function and param descriptions. Do this as you code so that you won't have to do it later.

Here is an example of what good documentation looks like

```
/**
* This method is used making logout API calls. 
*
* <p>Ref: 
* 1) https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletRequest.html
* 2) https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html
* 3) https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/ResponseEntity.html</p>
*
* @param request The [HttpServletRequest] request object. The servlet 
* container creates an HttpServletRequest object and passes it as an 
* argument to the servlet's service methods (doGet, doPost, etc).
* @param response The [HttpServletResponse] response object. Extends the 
* ServletResponse interface to provide HTTP-specific functionality in 
* sending a response. For example, it has methods to access HTTP headers 
* and cookies.
* @return Extension of HttpEntity that adds a HttpStatus status code. Used 
* in RestTemplate as well @Controller methods.
*/
```

All I did was copy/paste the description from the official docs. You can do the same. When you double click `HttpServletRequest` and then press Ctrl + Q, the official documentation shows up. Copy/paste the description.

This way it is very clear when reading the code what everything is, where you got your information from, ect. I include links for reference. It's all explained really clearly.